src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import classNames from 'classnames';
+import { UserInfo } from '../UserInfo/UserInfo';
+import { TodoInfo } from '../TodoInfo/TodoInfo';
+import './TodoList.scss';
+
+export type Props = {
+  todos: Todo[];
+};
+
+export const TodoList: React.FC<Props> = (props) => {
+  const { todos } = props;
+
+  return (
+    <ul className="list">
+      {todos.map(todo => (
+        <li
+          key={todo.id}
+          className={classNames(
+            'list__item',
+            { 'list__item--active': todo.completed },
+          )}
+        >
+          {todo.user && <UserInfo user={todo.user} />}
+          {todo.user && <TodoInfo todo={todo} />}
```

Comment:
```suggestion
          <TodoInfo todo={todo} />
```
Для чого нам тут ця перевірка?